tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -22,7 +22,7 @@ The audit logging feature is disabled by default. To audit a cluster, you need t
 ## Prerequisites
 
 - You are using a TiDB Dedicated cluster. Audit logging is not available for TiDB Serverless clusters.
-- You are the audit administrator of your organization in TiDB Cloud. Otherwise, you cannot see the audit-related options in the TiDB Cloud console. For more information, see [Manage role access](/tidb-cloud/manage-user-access.md#manage-role-access).
+- You are the `Project Owner` of your project in TiDB Cloud. Otherwise, you cannot see the database audit-related options in the TiDB Cloud console. For more information, see [User roles](/tidb-cloud/manage-user-access.md#user-roles).
```

Comment:
```suggestion
- If you are in the `Organization Owner` or `Project Owner` role of your organization. Otherwise, you cannot see the database audit-related options in the TiDB Cloud console. For more information, see [User roles](/tidb-cloud/manage-user-access.md#user-roles).
```